File: reader/DNASuitEdge.cpp
#include "DNASuitEdge.h"

DNASuitEdge::DNASuitEdge() {};
/*DNASuitEdge::DNASuitEdge(unsigned short start_point, unsigned short end_point,
						   unsigned short zone_id) : m_start_point(start_point),
						   m_end_point(end_point), m_zone_id(zone_id);*/

DNASuitEdge::~DNASuitEdge() {};

void DNASuitEdge::set_start_point(unsigned short start_point)
{
	m_start_point = start_point;
};

void DNASuitEdge::set_end_point(unsigned short end_point)
{
	m_end_point = end_point;
};

void DNASuitEdge::set_zone_id(unsigned short zone_id)
{
	m_zone_id = zone_id;
};

unsigned short DNASuitEdge::get_start_point()
{
	return m_start_point;
};

unsigned short DNASuitEdge::get_end_point()
{
	return m_end_point;
};

unsigned short DNASuitEdge::get_zone_id()
{
	return m_zone_id;
};

File: libpandadna/DNASuitEdge.py
class DNASuitEdge:
    COMPONENT_CODE = 22

    def __init__(self, startPoint, endPoint, zoneId):
        self.startPoint = startPoint
        self.endPoint = endPoint
        self.zoneId = zoneId

    def setStartPoint(self, startPoint):
        self.startPoint = startPoint

    def setEndPoint(self, endPoint):
        self.endPoint = endPoint

    def setZoneId(self, zoneId):
        self.zoneId = zoneId

    def getStartPoint(self):
        return self.startPoint
        
    def getEndPoint(self):
        return self.endPoint
        
    def getZoneId(self):
        return self.zoneId
        